art/dashboard/app.py

```diff
@@ -142,40 +26,56 @@ def build_timeline():
     Input("dropdown-selection", "value"),
 )
 def updateTable(step_name):
-    samples = []
-    for status in ["succesfull", "failed"]:
-        for sample in steps[step_name]["results"][status]:
-            samples.append({**sample, "status": status})
-    df = pd.DataFrame(samples)
+    df = OUTER_DFS[step_name]
+    df.successfull = df.successfull.astype(int)
 
     conditional = [
         {
             "if": {
-                "filter_query": "{status} = succesfull",
+                "filter_query": "{successfull} = 1",
```

Comment:
Could we have true/false instead of 1/0 here?

Reply:
dash seem not to support this xD I tried